examples/artistDetails/Ballerina.toml

```
@@ -0,0 +1,8 @@
+[package]
+org = "wso2"
+name = "artistDetails"
```

Let's follow the Ballerina package naming convention 
```suggestion
name = "artist_details"
```

---

Similarly we can change the directory name as `artist-details `